src/FakeItEasy/ExceptionMessages.resx

```diff
@@ -208,4 +208,7 @@ The following constructors failed:
   <data name="FakeManagerWasInitializedWithDifferentProxyMessage" xml:space="preserve">
     <value>The fake manager was initialized for a different proxy.</value>
   </data>
+  <data name="NotRecognisedAsFake" xml:space="preserve">
+    <value>{0} is not recognized as a fake object.</value>

```

Comment:
This produces messages like "System.Object is not recognized as a fake object.", which makes it sound like we passed a `Type` object whose value is `System.Object` into `GetFakeManager`. 

Perhaps better would be to indicate that "Object of type System.Object is not recognized as a fake object."